Tella/Data/Networking/Repositories/UwaziServerRepositories.swift

```diff
@@ -13,42 +13,59 @@ struct UwaziServerRepository: WebRepository {
 
     func login(username: String,
                password: String,
-               serverURL: String) -> AnyPublisher<(UwaziLoginResult,HTTPURLResponse?), APIError> {
-
-        let call : AnyPublisher<(UwaziLoginResult,HTTPURLResponse?), APIError> = callReturnsHeaders(endpoint: API.login((username: username, password: password, serverURL: serverURL)))
-
-        return call
+               serverURL: String) -> AnyPublisher<(BoolResponse,String?), APIError> {
```

Comment:
@rob-horizontal-iOS I think it's better to update the return result of the publisher, so we can use only the token and  BoolResponse is unuseful

Reply:
Just used the `String?`

Reply:
@rob-horizontal-iOS yes, because I think when the token is null, it will be an error and I I it's better than a `ServerTokenType`